neural_modelling/src/neuron/plasticity/stdp/synapse_structure/synapse_structure_weight_impl.h

```diff
@@ -4,8 +4,11 @@
 //---------------------------------------
 // Structures
 //---------------------------------------
-// Plastic synapse types are just weights;
-typedef weight_t plastic_synapse_t;
+// Plastic synapse types have weights and eligibility traces
+typedef struct Plastic_synapses {
+    int16_t eligibility_trace;
+    weight_t weight;
+} plastic_synapse_t;
```

Comment:
The performance of accessing these will probably be massively improved if you switch from structs to ``int16_t`` in the same way I did in this change https://github.com/SpiNNakerManchester/sPyNNaker/commit/97071e4d2b99a1ad17ba913c7de039a9967d1d43